screenpy/actions/attach_the_file.py

```diff
@@ -31,7 +31,8 @@ def perform_as(self, _: Actor) -> None:
         """Direct the Narrator to attach a file."""
         the_narrator.attaches_a_file(self.filepath, **self.attach_kwargs)
 
-    def __init__(self, filepath: str, **kwargs: Any) -> None:
+    # ANN401 ignored here to allow for new adapters to use any kwargs.
+    def __init__(self, filepath: str, **kwargs: Any) -> None:  # noqa: ANN401
```

Comment:
Do you think we can use `object` here?

Reply:
I'm not sure i fully understand when it's best to use `object` over `Any`, but all the times i've tried have ended in disaster.